Mathlib/Analysis/Normed/Field/Basic.lean

```diff
@@ -1095,6 +1096,19 @@ instance toNormedCommRing [NormedCommRing R] [SubringClass S R] (s : S) : Normed
 
 end SubringClass
 
+namespace SubfieldClass
+
+variable {S F : Type*} [SetLike S F]
+
+/--
+If `s` is a subfield of a normed field `F`, then `s` is equipped with an induced normed
+field structure.
+-/
+instance toNormedField [NormedField F] [SubfieldClass S F] (s : S) : NormedField s :=
```

Comment:
```suggestion
instance instNormedField [NormedField F] [SubfieldClass S F] (s : S) : NormedField s :=
```
the `to` names are usually reserved for `Foo S -> Bar S`, but this is `[Foo S] (s : S) : Bar s` which is different.

Reply:
Ah, I guess this matches the above though!